get-started/graph-db-vs-nosql/graph-db-vs-nosql.adoc

```diff
@@ -10,43 +10,43 @@
 
 .Goals
 [abstract]
-This guide quickly explains how graph databases are related to other NoSQL databases.
+This guide quickly explains how graph databases are related to other NoSQL databases and how they differ.
 
 .Prerequisites
 [abstract]
-You need no prior knowledge of other NoSQL stores, although it might be helpful to have read the guide on link:/developer/get-started/graph-database[graph databases].
+You need no prior knowledge of other NoSQL stores, although it is helpful to have read the guide on link:/developer/get-started/graph-database[graph databases] and understand basic data modeling questions and concepts.
 
 [role=expertise]
 {level}
 
 toc::[]
 
+
+== NoSQL Databases
+
 Although unhelpfully named, the NoSQL ("Not only SQL") space brings together many interesting solutions offering different data models and database systems, each suitable for different use cases and shapes of data.
 
 With the advent of the NoSQL movement, the "one-size-fits-all" proposition of large relational systems was replaced by making conscious decisions about finding the right tool for the job.
 
-Most NoSQL systems are *aggregate oriented*, providing only atomic operations within their natural aggregate, which forms only one dedicated view of your data.
-Other projections have to be computed crunching or duplicating your data.
-Focusing on one aggregate at a time allows for easy sharding of bucketed data across a network of machines along the aggregate dimension, for instance the *Document* in Document Databases.
+++++
+<iframe width="680" height="425" src="https://www.youtube.com/embed/5Tl8WcaqZoc" frameborder="0" allowfullscreen></iframe>
+++++
 
-Graph databases handle fine-grained networks of information providing *any perspective* on your data that fits your use-cases.
-Known and trusted from relational systems, transactional guarantees protect updates of that connected data conforming to ACID standards.
+Most NoSQL systems are *aggregate-oriented*, providing only atomic operations within their natural aggregate, which forms only one dedicated view of your data.
```

Comment:
Make these two sentences (35, 36) clearer. What are atomic operations? What's a natural aggregate? What's a dedicated view? Sharding? Projections? 

Maybe just use simpler words, with little prior knowledge of other NoSQL systems I'm a bit lost.

Reply:
Yep, makes sense!